bcda/auth/client/ssas_test.go

```diff
@@ -273,12 +273,13 @@ func (s *SSASClientTestSuite) TestGetToken() {
 		s.FailNow(constants.CreateSsasErr, err.Error())
 	}
 
-	respKey, err := client.GetToken(authclient.Credentials{ClientID: "happy", ClientSecret: "client"})
+	respKey, respExp, err := client.GetToken(authclient.Credentials{ClientID: "happy", ClientSecret: "client"})
```

Comment:
can we have this happy path test case return respExp with a value, that we can then assert as populated? (would need to update line 260 to have w.Write include the new expires_in with sample value used for testing)